db/migrate/20220919153449_create_tulsa_blotter_inmates.rb

```diff
@@ -0,0 +1,21 @@
+class CreateTulsaBlotterInmates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :inmates do |t|
```

Comment:
we'll want to namespace the tables here
so this should be tulsa_blotter_inmates